python/sglang/srt/lora/utils.py

```diff
@@ -96,6 +96,7 @@ def get_normalized_lora_weight_names(
         "v_proj": "qkv_proj",
         "gate_proj": "gate_up_proj",
         "up_proj": "gate_up_proj",
+        "embed_tokens": (["embed_tokens"], ["embed_tokens"]),
```

Comment:
Is this a merge error. Can you make sure your code is merged with the latest main and can pass basic LoRA CI. Thanks!